contracts/src/v0.8/l2ep/dev/scroll/ScrollSequencerUptimeFeed.sol

```diff
@@ -68,6 +70,10 @@ contract ScrollSequencerUptimeFeed is
   /// @param l2CrossDomainMessengerAddr Address of the L2CrossDomainMessenger contract
   /// @param initialStatus The initial status of the feed
   constructor(address l1SenderAddress, address l2CrossDomainMessengerAddr, bool initialStatus) {
+    if (l2CrossDomainMessengerAddr == address(0)) {
+      revert ZeroAddress();
```

Comment:
I would expect a test for each revert. Since this adds a revert I would expect a test. Same for the other one